projects/1.20/assets/interiors/interiors/lang/zh_cn.json

```
@@ -0,0 +1,33 @@
+{
+    "creativetab.extended_seating_tab": "机械动力：内饰",
+    "creativetab.extended_seating": "机械动力：内饰",
+    "block.interiors.kelp_seat": "海带坐垫",
+    "block.interiors.kelp_chair": "海带座椅",
+    "block.interiors.seatwood_planks": "座椅木板",
+
+    "block.interiors.white_chair": "白色座椅",
+    "block.interiors.light_gray_chair": "淡灰色座椅",
+    "block.interiors.gray_chair": "灰色座椅",
+    "block.interiors.black_chair": "黑色座椅",
+    "block.interiors.brown_chair": "棕色座椅",
+    "block.interiors.red_chair": "红色座椅",
+    "block.interiors.orange_chair": "橙色座椅",
+    "block.interiors.yellow_chair": "黄色座椅",
+    "block.interiors.lime_chair": "黄绿色座椅",
+    "block.interiors.green_chair": "绿色座椅",
+    "block.interiors.cyan_chair": "青色座椅",
+    "block.interiors.light_blue_chair": "淡蓝色座椅",
+    "block.interiors.blue_chair": "蓝色座椅",
+    "block.interiors.purple_chair": "紫色座椅",
+    "block.interiors.magenta_chair": "品红色座椅",
+    "block.interiors.pink_chair": "粉红色座椅",
+    
+    "block.interiors.chair.tooltip": "CHAIR",
+    "block.interiors.chair.tooltip.summary": "坐下来，享受更好的旅程！将玩家固定在移动装置上。当作家具也不错！有多种颜色可供选择。",
+    "block.interiors.chair.tooltip.condition1": "右击座椅",
```

```suggestion
    "block.interiors.chair.tooltip.condition1": "右击座椅时",
```
这里的显示格式应该是
```
右击座椅时
  blahblahblah
对座椅使用扳手时
  blehblehbleh
```
需要进游戏验证

---

实际进游戏看了一下，并没有出现。
应该是没做

---

怪，没做可还行（）